selfdrive/car/toyota/fingerprints.py

```diff
@@ -823,6 +824,7 @@
       b'\x03896634760100\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00',
       b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00',
       b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701004\x00\x00\x00\x00',
+      b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00????????????????',
```

Comment:
this is odd

Reply:
Previous version used to be `b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00'`, and the user's EPS version bumped slightly at the same time causing it to fuzzy fingerprint: `b'8965B47023\x00\x00\x00\x00\x00\x00'` -> `b'8965B47070\x00\x00\x00\x00\x00\x00'` + the VIN stopped returning.

So maybe a bad ECU update from a dealer? The strange part is that the user is not at any kind of auto shop between these two routes as far as I can tell

723b102eb5d8291e/2024-02-21--11-05-51
723b102eb5d8291e/2024-02-21--15-22-57

Reply:
The Prius seems to have recovered. Will add the check next time we see this.